### DISCUSSION TITLE
A Politico journalist got banned from Bluesky for seemingly no reason and without explanation

### DISCUSSION THREAD

**Discussion by venteto** [bluesky-social/atproto#3241] [General Discussion]
Is Bluesky really just going to be (or already is) more of the same, the same old same old of all social media? It seems no high-minded visions of protocols or anything else is actually preventing history from repeating:

https://www.politico.com/news/magazine/2024/12/12/bluesky-twitter-x-alternative-toxic-00193862

**Comment by grasshoppermouse** [comment]
I don't know if this is why, but see this for one possible explanation: https://bsky.app/profile/safety.bsky.app/post/3ld7e2hs7622r
